test/e2e/tinkerbell_test.go

```diff
@@ -54,12 +54,12 @@ func TestTinkerbellKubernetes123BottleRocketAWSIamAuth(t *testing.T) {
 	runTinkerbellAWSIamAuthFlow(test)
 }
 
-func TestTinkerbellKubernetes125BottleRocketAWSIamAuth(t *testing.T) {
+func TestTinkerbellKubernetes126BottleRocketAWSIamAuth(t *testing.T) {
```

Comment:
Do we want none of tests for 125? I noticed that there is currently still one for 123

Reply:
There isn't really a set protocol. We update most of the tests to use the latest version, but we also leave a few tests with older versions just to have insight and coverage of the behavior with other versions, just to ensure that we aren't introducing breaking changes for older versions.